src/addons/addon.cpp

```diff
@@ -68,7 +68,7 @@ QList<ConditionCallback> s_conditionCallbacks{
      [](Addon* addon, const QJsonValue& value) -> AddonConditionWatcher* {
        QStringList features;
        for (const QJsonValue& v : value.toArray()) {
-         features.append(v.toString().toLower());
```

Comment:
Should we add some tests to prevent this from happening again?

Reply:
good point. We have tests for the condition watcher but not connected to the manifest parser.